help/en/html/FAQ.shtml

```diff
@@ -87,115 +87,115 @@
     
 
       <dl class="faq">
-        <dt class="on" id="canuse">Can I use JMRI to...</dt>
+        <dt class="on" id="canuse"><strong>Can I use JMRI to...</strong></dt>
```

Comment:
With the "strong" here, the line no long goes from regular to bold when the paragraph is expanded on the web page.  Suggest not adding "strong" around dt lines for each FAQ.